scripts/globals/status.lua

```diff
@@ -893,6 +893,7 @@ xi.effectFlag =
     OFFLINE_TICK    = 0x2000000,
     AURA            = 0x4000000,
     HIDE_TIMER      = 0x8000000,
+    ON_ZONE_PATHOS  = 0x10000000,
```

Comment:
Might want to front pad the zeroes on the previous flags so they're aligned

Reply:
+1